docs/source/components/nodes/tof.rst

```diff
@@ -55,10 +129,27 @@ Usage
     pipeline = dai.Pipeline()
 
     tof_cam = pipeline.create(dai.node.Camera)
+    # Decoded depth FPS will be /2 due to shuffle/non-shuffle averaging
```

Comment:
We will likely chnage this behavior to x2 in FW by default. Should be merged today